braph2genesis/graphs/_OrdMxBUD.gen.m

```diff
@@ -0,0 +1,203 @@
+%% ¡header!
+OrdMxBUD < OrdMxWU (g, binary undirected multiplex with fixed densities) is a ordinal multiplex binary undirected with fixed densities.
+
+%%% ¡description!
+In a ordinal binary undirected multiplex with fixed densities (BUD), the layers are 
+ those of binary undirected (BU) multiplex graphs derived from the same 
+ weighted supra-connectivity matrices binarized at different densities.
+The supra-connectivity matrix has a number of partitions equal to the number of densities.
+The layers are connected in an ordinal fashion, i.e., only consecutive layers are connected.
```

Comment:
There is just **one supra-adjacency matrix** in the weighted multiplex/multilayer, if you talk in plural you mean the within-layer connectivity matrix. Also talk about the supra-adjacency matrix, not the supra connectivity matrix.

Reply:
Also: In an ordinal... ; this needs to be rewritten.